.build/BuildHelper/Get-Version.ps1

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+<#
+    .SYNOPSIS
+    Gets the current version.
+
+    .DESCRIPTION
+    Run this from the /src/scripts folder.
+
+    .PARAMETER Template
+    Optional. Name of the template or module to package. Default = * (all).
+
+    .PARAMETER Build
+    Optional. Indicates whether the Build-Toolkit command should be executed first. Default = false.
+
+    .EXAMPLE
+    ./Package-Toolkit
+
+    Generates ZIP files for each template using an existing build.
+
+    .EXAMPLE
+    ./Package-Toolkit -Build
+
+    Builds the latest code and generates ZIP files for each template.
+#>
+function Get-Version {
+    param
+    (
+        [switch]
+        $Major,
+
+        [switch]
+        $Minor,
+
+        [switch]
+        $Patch,
+
+        [switch]
+        $Prerelease,
+    
+        [AllowEmptyString()]
+        [AllowNull()]
+        [ValidateSet($null, '', 'dev', 'alpha', 'preview')]
+        [string]
+        $Label = 'dev'
+    )
+    
+    $update = if ($Major) { "major" } elseif ($Minor) { "minor" } elseif ($Patch) { "patch" }
+    $ver = $null
+    if ($update) {
+        $ver = npm --no-git-tag-version version $update
```

Comment:
node.js needs to be installed as part of the pre-requisites for this to work

Reply:
Updated Get-Version to not require NodeJS